policyengine_uk/parameters/gov/dfe/extended_childcare_entitlement/expense_rate.yaml

```
@@ -0,0 +1,28 @@
+description: The Department for Education sets hourly rates to convert childcare hours into expenses for extended childcare entitlement. These rates vary by child's age and are used to calculate the funding provided to childcare providers.
```

Use `this/these` in descriptions
```suggestion
description: The Department for Education converts childcare hours into expenses for extended childcare entitlement at these rates, depending on the child's age.
```

---

Is there a clear style guide we have that would imply this change

---

It was not obvious to me to suggest this!